### ISSUE TITLE
Detection results trainval

### ISSUE THREAD

**Issue by Jan-Nico** [tianweiy/CenterPoint#49]
Hi, thank you for the nice work!
Would it be possible to get the detection results of your method also for the trainval set of nuScenes?

**Comment by chowkamlee81** [comment]
All the model results are there in modelzoo https://github.com/tianweiy/CenterPoint/blob/master/docs/MODEL_ZOO.md
under Nuscenes_3d tracking in json format

**Comment by tianweiy** [comment]
Yeah, you can find val set result in the Google drive. The train set is too large to upload for now and you can easily generate it yourselves.
